cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala

```diff
@@ -214,7 +214,7 @@ object SparkMainWrapper {
           executor.awaitTermination(5L, TimeUnit.SECONDS)
           // Send the complete call
           client.completed(workflowToken)
-          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
+          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun:Any)
```

Comment:
"Any" forces Scala to choose one between the two methods. Here is the error: 
build	27-Apr-2017 19:12:07	[ERROR] /var/bamboo/xml-data/build-dir/CDAP-DRC-JOB1/cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala:217: error: ambiguous reference to overloaded definition,
build	27-Apr-2017 19:12:07	[ERROR] both method info in trait Logger of type (x$1: String, x$2: <repeated...>[Object])Unit
build	27-Apr-2017 19:12:07	[ERROR] and  method info in trait Logger of type (x$1: String, x$2: Any, x$3: Any)Unit
build	27-Apr-2017 19:12:07	[ERROR] match argument types (String,String,String)
build	27-Apr-2017 19:12:07	[ERROR]           LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
build	27-Apr-2017 19:12:07	[ERROR]               ^
build	27-Apr-2017 19:12:08	[ERROR] one error found